src/odemis/driver/autoscript_client.py

```diff
@@ -33,7 +33,7 @@
 import pkg_resources
 import Pyro5.api
 from Pyro5.errors import CommunicationError
-
+from scipy import ndimage
 from odemis import model, util
```

Comment:
Can you follow the import order from the guidelines? https://github.com/delmic/odemis/blob/master/doc/develop/coding-guidelines.rst#import-order
```suggestion
from Pyro5.errors import CommunicationError
from scipy import ndimage

from odemis import model, util
```